jsonnet/jsonnetfile.lock.json

```diff
@@ -1,7 +1,17 @@
 {
+  "version": 1,
   "dependencies": [
     {
-      "name": "etcd-mixin",
+      "source": {
```

Comment:
This is because of the v0.3.1 bump of jb, should we be introducing this to 4.4?

Reply:
ping @paulfantom

Reply:
fyi: the out of band consensus is that we use up-to-date jsonnet-bundler for backports and start pinning from the next sprint.

Reply:
Yes we agreed this is fine.